### PR TITLE
changing pod from mParticle-Swrve to Swrve-mParticle

### PR DESCRIPTION
**Pull request by cooneycj**
requested change from mParticle-Swrve to Swrve-mParticle.

renaming all instances in the code of that order.

@adam-govan could you have a look at this to see if it looks ok before I merge?

### CODE REVIEW

**Comment by cooneycj** [comment]
@adam-govan bumping this thread

File: Swrve-mParticle.podspec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Pod::Spec.new do |s|
-    s.name             = "mParticle-Swrve"
+    s.name             = "Swrve-mParticle"
     s.version          = "1.0"
```
Comment (adam-govan) [inline Swrve-mParticle.podspec]: ~~ version number should be 0.1.0 as discussed

Comment (cooneycj) [reply Swrve-mParticle.podspec]: done

**Comment by cooneycj** [comment]
@adam-govan everything else look ok?

**Pull request merged**